app/javascript/components/booking/Bookings.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { fetchBookings } from "../redux/bookings";
+import { fetchDevelopers } from "../redux/developer";
+
+function Bookings() {
+  const dispatch = useDispatch();
+  const bookings = useSelector((state) => state.bookings.bookings);
+  const developers = useSelector((state) => state.developers.developers);
+  useEffect(() => {
+    dispatch(fetchBookings());
+    dispatch(fetchDevelopers());
+  }, []);
+  console.log(developers);
```

Comment:
- [ ] Kindly remove this old piece of debugging code in order to improve your app's cleanlines.